src/app/tests/suites/ciTests.json

```diff
@@ -284,15 +284,12 @@
         "DL_LockUnlock",
         "DL_Schedules",
         "Test_TC_DRLK_1_1",
-        "Test_TC_DRLK_2_2",
-        "Test_TC_DRLK_2_3",
```

Comment:
Again, after https://github.com/project-chip/connectedhomeip/pull/28526 you could re-enable this in CI.  Maybe a tracking issue?